### DISCUSSION TITLE
The download of Foundry failed. Here is a screenshot of the error message. What should I do

### DISCUSSION THREAD

**Discussion by xyl2004** [Cyfrin/foundry-full-course-cu#2584] [Q&A]
Here is code of the error message. What should I do？
,,,
curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   167  100   167    0     0    104      0  0:00:01  0:00:01 --:--:--   104
curl: (6) Could not resolve host: raw.githubusercontent.com
,,,

**Comment by EngrPips** [comment]
It might be an internet connection problem. Are you sure your internet connection is stable?
